tests/meta/test_ensure_type_hints.py

```diff
@@ -102,7 +102,7 @@ def test_check_get_without_id_function_type_hints(
             base_type=gitlab.mixins.GetWithoutIdMixin,
             class_info=class_info,
             method_template=GET_WITHOUT_ID_METHOD_TEMPLATE,
-            optional_return=True,
+            optional_return=False,
```

Comment:
question: do you think this argument is still needed? I don't think we'll be returning `None` from `get()` at any point from now on.

Reply:
not too important :)